scripts/ingestors/madis/to_iemaccess.py

```
@@ -36,18 +37,18 @@ def find_file(variant, valid: datetime):
         else:
             uri = f"{WEBROOT}LDAD/{variant[:-1]}/netCDF/{valid:%Y%m%d_%H}00.gz"
         try:
+            resp = requests.get(uri, timeout=30)
+            resp.raise_for_status()
             # Prevent script errors from filling up /tmp
             with open("/tmp/madis_download.gz", "wb") as tmp:
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp/madis_download.gz"

(S108)